src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -193,13 +182,24 @@ open class ModellingGroupController(context: ControllerContext)
         val id = estimateRepository.addBurdenEstimateSet(
                 path.groupId, path.touchstoneId, path.scenarioId,
                 data,
-                uploader = context.username!!,
+                uploader = context.username ?: "testing",
```

Comment:
Where is this "testing" username needed? Can it be provided via a mock instead of in the code?

Reply:
Oops, that's just a mistake left in from testing. Good catch!